tests/check_framework/test_model_checks.py

```diff
@@ -403,6 +403,29 @@ class Child(Parent):
 
         self.assertEqual(checks.run_checks(app_configs=self.apps.get_app_configs()), [])
 
+    def test_explicit_pk_with_abstract_parent(self):
+        class Abstract(models.Model):
+            class Meta:
+                abstract = True
+
+        class Child(Abstract):
+            id = models.AutoField(primary_key=True)
+
```

Comment:
This is unused now, besides that, it's looking good. However, I presume one of the follows has more ideas on how to improve the tests. They know the test suite better.